### PR TITLE
docs: revised the performance documentation

### PR DESCRIPTION
**Pull request by nolouch**
Signed-off-by: nolouch <nolouch@gmail.com>

### First-time contributors' checklist 

- [ ] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### What is changed, added or deleted? (Required)

fix the config

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [x] v5.4 (TiDB 5.4 versions)
- [x] v5.3 (TiDB 5.3 versions)
- [x] v5.2 (TiDB 5.2 versions)
- [x] v5.1 (TiDB 5.1 versions)
- [x] v5.0 (TiDB 5.0 versions)
- [x] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

https://github.com/pingcap/docs-cn/issues/6466

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: tune-tikv-memory-performance.md [hunk 1]
```diff
@@ -101,13 +101,17 @@ job = "tikv"
 # 如果机器上有多块磁盘，可以将 Raft RocksDB 的数据放在不同的盘上，提高 TiKV 的性能。
 # raftdb-path = "/tmp/tikv/store/raft"
 
-region-max-size = "384MB"
-# Region 分裂阈值
-region-split-size = "256MB"
 # 当 Region 写入的数据量超过该阈值的时候，TiKV 会检查该 Region 是否需要分裂。为了减少检查过程
 # 中扫描数据的成本，数据过程中可以将该值设置为32MB，正常运行状态下使用默认值即可。
```
Comment (TomShawn) [inline tune-tikv-memory-performance.md:105]: 这里的“数据过程”是“导入数据的过程”吗？

**Review by Connor1996 — APPROVED** [review]
LGTM

File: tune-tikv-memory-performance.md (hunk 1 above)
Comment (nolouch) [reply tune-tikv-memory-performance.md:105]: 导入数据和正常写入都可以，只要 region 的数据增长了

Comment (TomShawn) [reply tune-tikv-memory-performance.md:105]:
```suggestion
# 中扫描数据的成本，导入数据过程中可以将该值设置为32MB，正常运行状态下使用默认值即可。
```

File: tune-tikv-memory-performance.md [hunk 2]
```diff
@@ -101,13 +101,17 @@ job = "tikv"
...
-region-split-size = "256MB"
 # 当 Region 写入的数据量超过该阈值的时候，TiKV 会检查该 Region 是否需要分裂。为了减少检查过程
-# 中扫描数据的成本，数据过程中可以将该值设置为32MB，正常运行状态下使用默认值即可。
+# 中扫描数据的成本，导入数据过程中可以将该值设置为 32 MB，正常运行状态下使用默认值即可。
 region-split-check-diff = "32MB"
 
+[coprocessor]
+
+## 当 Region [a,e) 的大小超过 `region_max_size`, 这个 Region 就会尝试进行分裂，比如分裂成[a,b),
+## [b,c), [c,d), [d,e) 后，这些 Region [a,b), [b,c), [c,d) 的大小会是 `region_split_size` (或者稍大于）
```
Comment (TomShawn) [inline tune-tikv-memory-performance.md:111]:
```suggestion
## 当区间为 [a,e) 的 Region 的大小超过 `region_max_size`，TiKV 会尝试分裂该 Region，例如分裂成 [a,b)、[b,c)、[c,d)、[d,e) 等区间的 Region 后
## 这些 Region [a,b), [b,c), [c,d) 的大小为是 `region_split_size` (或者稍大于 `region_split_size`）
```

**Review by TomShawn — APPROVED** [review]
LGTM

**Pull request merged**